tests/js/common/aql/aql-arangosearch-inverted-index-search-alias-community.js

```diff
@@ -0,0 +1,971 @@
+                },
+                {
+                    'type': 'inverted', 'name': 'nest2', // empty name
+                    'fields': [
+                        {
+                            "expression": "RETURN SPLIT(@param, '.') ",
+                            "name": "",
+                            "features": [
+                                "norm",
+                                "frequency"
+                            ],
+                            "override": false,
+                            "includeAllFields": true,
+                            "trackListPositions": true
+                        }
+                    ]
+                }
+            ];
+
+            db._create("failCollection");
+            for (let i = 0; i < failDefinitions.length; i++) {
+                let d = failDefinitions[i];
+                try {
+                    db.failCollection.ensureIndex(d);
+                    assertTrue(false);
+                } catch (e) {
+                    assertEqual(errors.ERROR_BAD_PARAMETER.code, e.errorNum);
+                    assertEqual(db.failCollection.indexes().length, 1);
+                }
+            }
+        },
+
+        testInvertedIndexRegressions: function () {
+
+            var testColl = db._create("testColl", { replicationFactor: 1, writeConcern: 1, numberOfShards: 1 });
+            for (let i = 0; i < 5; i++) {
+                let cv_field = String(i) + String(i + 1);
+                testColl.save({ a: "foo", b: "bar", c: i, cv_field: cv_field });
+                testColl.save({ a: "foo", b: "baz", c: i, cv_field: cv_field });
+                testColl.save({ a: "bar", b: "foo", c: i, cv_field: cv_field });
+                testColl.save({ a: "baz", b: "foo", c: i, cv_field: cv_field });
+            }
+
+            // SEARCH-353
+            {
+                assertInvertedIndexCreation("testColl", { name: "i1", type: "inverted", fields: ["a"] });
+
+                assertInvertedIndexCreation("testColl", { name: "i2", type: "inverted", fields: ["b"] });
+                assertEqual(testColl.indexes().length, 3);
+            }
+
+            // SEARCH-346
+            {
+                assertInvertedIndexCreation("testColl", { type: "inverted", name: "i3", fields: ["cv_field"] });
+                assertInvertedIndexCreation("testColl", { type: "inverted", name: "i4", fields: ["cv_field"] }, false);
+                // sync indexes
+                db._query("for d in testColl OPTIONS {'indexHint': 'i1', 'forceIndexHint': true, 'waitForSync': true} filter d.a == 'foo' collect with count into c return c");
+                db._query("for d in testColl OPTIONS {'indexHint': 'i2', 'forceIndexHint': true, 'waitForSync': true} filter d.b == 'bar' collect with count into c return c");
+                db._query("for d in testColl OPTIONS {'indexHint': 'i3', 'forceIndexHint': true, 'waitForSync': true} filter d.cv_field == 'cv_field' collect with count into c return c");
```

Comment:
Why without i4?

Reply:
Because `i4` is not created. It has the same definition as `i3`. And I check explicitly that this index is not creating